tokio/src/blocking.rs

```diff
@@ -45,16 +48,16 @@ cfg_not_rt! {
         type Output = Result<R, std::io::Error>;
 
         fn poll(self: Pin<&mut Self>, _cx: &mut Context<'_>) -> Poll<Self::Output> {
-            unreachable!()
```

Comment:
Hmm, I think the `unreachable` here is still helpful when this code path is triggered by bugs.

Reply:
I think we can add a simple comment for this `unreachable` instead of declaring a new type.